types/storybook__addon-knobs/storybook__addon-knobs-tests.tsx

```diff
@@ -75,7 +75,7 @@ stories.add('with all knobs', () => {
 
   return (
     <div style={style}>
-      I'm {name} and I was born on "{dob}"
+      I'm {name} and I was born on "{dob.toString()}"
```

Comment:
This _could_ be fixed by adding `{ toString(): string }` to `ReactText`, but it's probably best not to get into _that_ rabbit hole... and it'd get us right back to accidentally accepting functions even when they're not acceptable children.